dpgen/generator/arginfo.py

```diff
@@ -101,6 +101,30 @@ def training_args() -> List[Argument]:
 
 
 # Exploration
+def model_devi_jobs_template_args() -> List[Argument]:
+    doc_template = 'Give a template for 01.model_devi.'
+    doc_template_lmp = 'The path to input.lammps template'
+    doc_template_plm = 'The path to input.plumed template'
+
+    args = [
+        Argument("lmp", str, optional=True, doc=doc_template_lmp),
+        Argument("plm", str, optional=True, doc=doc_template_plm),
+    ]
+    return Argument("template", list, args, [], optional=True, repeat=True, doc=doc_template)
+
+
+def model_devi_jobs_rev_mat_args() -> List[Argument]:
+    doc_rev_mat = '' # looking forward to update
+    doc_rev_mat_lmp = '' # looking forward to update
+    doc_rev_mat_plm = '' # looking forward to update
+
+    args = [
+        Argument("lmp", dict, optional=True, doc=doc_rev_mat_lmp),
+        Argument("plm", dict, optional=True, doc=doc_rev_mat_plm),
+    ]
+    return Argument("rev_mat", list, args, [], optional=True, repeat=True, doc=doc_rev_mat)
```

Comment:
After review, I think both `template` and `rev_mat` are not repeat. Instead they are simple `dict`s.
https://github.com/deepmodeling/dpgen/blob/5ae6b2f70d191e20ba0794bed2ef03d9e4162f1f/dpgen/generator/run.py#L705
https://github.com/deepmodeling/dpgen/blob/5ae6b2f70d191e20ba0794bed2ef03d9e4162f1f/dpgen/generator/run.py#L687